app/views/certificates/upload.html.erb

```diff
@@ -0,0 +1,42 @@
+<fieldset class="govuk-fieldset">
+  <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+    <h1 class="govuk-fieldset__heading">Upload your certificate</h1>
+  </legend>
+
+  <div class="govuk-form-group">
+    <%= form_for @upload, url: '/upload'  do |f| %>
+      <h3 class="govuk-heading-m">Certificate usage</h3>
+      <div class="govuk-radios govuk-radios--inline">
+        <div class="govuk-radios__item">
+          <%= f.radio_button :usage, 'Signing', class: 'govuk-radios__input'  %>
+          <%= f.label :usage, 'Signing', :value => 'Signing', class: 'govuk-label govuk-radios__label' %>
```

Comment:
I would use lowercase value, i.e. `signing`

Reply:
Is there any need for the value under label?